ui/src/message_popup/data_lists.jsx

```diff
@@ -0,0 +1,142 @@
+    id: 1,
+    name: 'Kevin',
+    grade: '7th grade',
+    gender: 'male',
+    race: 'Hispanic',
+    behavior: 'Disruptive in class',
+    ell: null,
+    learningDisabilities: null,
+    academicPerformance: 'Grades are C/D and borderline failing. Failed science the last two semesters. Reading and writing is at the 5th grade level. Arrives late to class most of the time.',
+    interests: null,
+    familyBackground: null,
+    ses: null,
+  },
+  {
+    id: 2,
+    name: 'Floyd',
+    grade: '7th grade',
+    gender: 'male',
+    race: 'Caucasian',
+    behavior: 'Attendance issues',
+    ell: null,
+    learningDisabilities: 'ADHD',
+    academicPerformance: null,
+    interests: 'Wants to be a firefighter',
+    familyBackground: 'Divorced parents, younger brother',
+    ses: 'Free and reduced lunch'
+  },
+  {
+    id: 3,
+    name: 'Maia',
+    grade: '7th grade',
+    gender: 'female',
+    race: 'Caucasian',
+    behavior: 'Discipline report',
+    ell: null,
+    learningDisabilities: null,
+    academicPerformance: null,
+    interests: 'In theater and loves art',
+    familyBackground: 'Single mom, two olders sisters',
+    ses: null
+  },
+  { 
+    id: 4, 
+    name: 'Hayin',
+    grade: '7th grade',
+    gender: 'female',
+    race: 'Korean',
+    behavior: null,
+    ell: null,
+    learningDisabilities: null,
+    academicPerformance: null,
+    interests: 'Plays the flute, very involved in church, loves coding',
+    familyBackground: '1st generation parents, twin sister',
+    ses: 'Free and reduced lunch'
+    
+  },
+  {
+    id: 5,
+    name: 'Mike',
+    grade: '7th grade',
+    gender: 'male',
+    race: 'African American',
+    behavior: null,
+    ell: null,
+    learningDisabilities: null,
+    academicPerformance: null,
+    interests: 'Plays in a band, sings in the school chorus',
+    familyBackground: '1 older brother',
+    ses: null,
+  },
+  {
+    id: 6,
+    name: 'Jasmine',
+    grade: '7th grade',
+    gender: 'female',
+    race: 'African American',
+    behavior: 'Quiet',
+    ell: null,
+    learningDisabilities: null,
+    academicPerformance: null,
+    interests: 'Plays tennis and soccer',
+    familyBackground: 'Divorced parents, only child',
+    ses: 'Free and reduced lunch'
+  },
+  {
+    id: 7,
+    name: 'Miquel',
+    grade: '7th grade',
+    gender: 'male',
+    race: 'Latino',
+    behavior: 'Attendance issues',
+    ell: 'Yes, Spanish is native language',
+    learningDisabilities: null,
+    academicPerformance: null,
+    interests: 'Has an after school job',
+    familyBackground: '3 younger siblings',
+    ses: 'Free and reduced lunch'
+  },
+  {
+    id: 8,
+    name: 'Ada',
+    grade: '7th grade',
+    gender: 'female',
+    race: 'Haitian',
+    behavior: null,
+    ell: 'Yes, Haitian Creole is native language',
+    learningDisabilities: 'Auditory disability',
+    academicPerformance: null,
+    interests: 'High-achiever, teacher\'s pet. Wants to be a doctor, is in yearbook',
```

Comment:
In ES6 you can also use a backtick as a quote if you like, so then you don't have to use the backslash to escape:
```
`High-achiever, teacher's pet. Wants to be a doctor, is in yearbook`
```